addons/point_of_sale/static/src/app/printer/render_service.js

```diff
@@ -80,9 +80,13 @@ registry.category("services").add("renderer", renderService);
  */
 const applyWhenMounted = async ({ el, container, callback }) => {
     const elClone = el.cloneNode(true);
+    const sameClassElements = container.querySelectorAll(`.${el.className}`);
```

Comment:
`.${el.className}` isn't correct. If there are multiple class names in the element like "class1 class2", then your selector will become ".class1 class2" and it won't be found because the correct selector for my example ".class1.class2".

I'd suggest that you get the class names and join them with ".".
```suggestion
    const sameClassElements = container.querySelectorAll(`.${[...el.classList].join(".")}`);
```

The preceding "dot" is important.

But to be honest, I'd prefer hardcoding the class names using "render-container" and  "pos-receipt".